packages/calypso-products/src/products-list.ts

```diff
@@ -453,6 +457,26 @@ export const JETPACK_SITE_PRODUCTS_WITH_FEATURES: Record<
 		getProductId: () => 2049,
 		getStoreSlug: () => PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_YEARLY,
 	},
+	[ PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_YEARLY ]: {
+		product_name: translate( 'Add-on Storage (3TB)' ),
+		product_slug: PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_YEARLY,
+		type: PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_YEARLY,
+		term: TERM_MONTHLY,
```

Comment:
Should it be that instead?
```
term: TERM_ANNUALLY,
bill_period: PLAN_ANNUAL_PERIOD,
```

Same question for the entry below.

Reply:
Ack! Yes, absolutely. I just pushed an update to fix this for all annual add-on storage products.